visual-studio-dev-vm-O365/azuredeploy.json

```diff
@@ -0,0 +1,275 @@
+      "allowedValues": [
+        "VS-2015-Comm-AzureSDK-2.9-W10T-Win10-N",
+        "VS-2015-Comm-AzureSDK-2.9-WS2012R2",
+        "VS-2015-Comm-VSU2-AzureSDK-29-W10T-N-x64",
+        "VS-2015-Comm-VSU2-AzureSDK-29-WS2012R2",
+        "VS-2015-Community-AzureSDK-2.7-Cordova-Win8.1-N-x64",
+        "VS-2015-Community-AzureSDK-2.7-W10T-Win10-N",
+        "VS-2015-Community-AzureSDK-2.7-WS2012R2",
+        "VS-2015-Ent-AzureSDK-2.8-Cordova-Win8.1-N-x64",
+        "VS-2015-Ent-AzureSDK-2.9-WS2012R2",
+        "VS-2015-Ent-AzureSDK-29-W10T-Win10-N",
+        "VS-2015-Ent-VSU2-AzureSDK-2.9-W10-N",
+        "VS-2015-Ent-VSU2-AzureSDK-29-W10T-N-x64",
+        "VS-2015-Ent-VSU2-AzureSDK-29-WS2012R2",
+        "VS-2015-Enterprise-AzureSDK-2.7-Cordova-Win8.1-N-x64",
+        "VS-2015-Enterprise-AzureSDK-2.7-W10T-Win10-N",
+        "VS-2015-Enterprise-AzureSDK-2.7-WS2012R2",
+        "VS-2015-Pro-AzureSDK-2.8-Cordova-Win8.1-N-x64",
+        "VS-2015-Professional-AzureSDK-2.7-Cordova-Win8.1-N-x64",
+        "VS-2015-Professional-AzureSDK-2.7-W10T-Win10-N"
+      ],
+      "metadata": {
+        "description": "Which version of Visual Studio you would like to deploy"
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+      "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "officeVersion": {
+      "type": "string",
+      "defaultValue": "Office2016",
+      "allowedValues": [
+        "Office2016",
+        "Office2013"
+      ],
+       "metadata": {
+        "description": "Which version of Office would you would like to deploy"
+      }
+    },    
+    "setupOfficeScriptFileName": {
+      "type": "string",
+      "defaultValue": "DeployO365SilentWithVersion.ps1",
+      "metadata": {
+        "description": "PowerShell script name to execute"
+      }
+    }
+  },
+  "variables": {
+    "apiVersionStorage":"2015-06-15",
+    "apiVersionNetwork":"2015-06-15",
+    "apiVersionCompute":"2015-06-15",
+    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'visstudiovm')]",
+    "vnet01Prefix": "10.0.0.0/16",
+    "vnet01Subnet1Name": "Subnet-1",
+    "vnet01Subnet1Prefix": "10.0.0.0/24",
+    "vmImagePublisher": "MicrosoftVisualStudio",
+    "vmImageOffer": "VisualStudio",
+    "vmOSDiskName": "VMOSDisk",
+    "vmVnetID": "[resourceId('Microsoft.Network/virtualNetworks', 'Vnet01')]",
+    "vmSubnetRef": "[concat(variables('VMVnetID'), '/subnets/', variables('Vnet01Subnet1Name'))]",
+    "vmStorageAccountContainerName": "vhds",
+    "vmNicName": "[concat(parameters('vmName'), 'NetworkInterface')]",
+    "vmIP01Name": "VMIP01"
+  },
+  "resources": [
+    {
+      "name": "[variables('storageAccountName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "location": "[resourceGroup().location]",
```

Comment:
I think this is ok - if somone wants to replace this it's a simple search and replace - it's also common in other templates and inline w/ BP